snapshots/sample/app/src/main/kotlin/com/emergetools/snapshots/sample/ui/LocalePreviews.kt

```diff
@@ -0,0 +1,13 @@
+package com.emergetools.snapshots.sample.ui
+
+import androidx.compose.ui.tooling.preview.Preview
+
+@Preview(
+  name = "English",
+  locale = "en"
+)
+@Preview(
+  name = "German",
+  locale = "de"
+)
+annotation class LocalePreviews
```

Comment:
Added a new multi-preview in the main module and moved the existing one to the UI module. This should let us test that same/different modules are resolved properly.